app/package.json

```diff
@@ -4,20 +4,24 @@
   "private": true,
   "scripts": {
     "serve": "vue-cli-service serve",
+    "test:unit": "vue-cli-service test:unit",
+    "lint": "vue-cli-service lint --max-warnings 0 --no-fix",
     "build:production": "vue-cli-service build --mode production",
     "build:staging": "vue-cli-service build --mode staging",
-    "lint": "vue-cli-service lint --max-warnings 0 --no-fix",
-    "lint:fix": "vue-cli-service lint --max-warnings 0",
-    "test:unit": "vue-cli-service test:unit"
+    "i18n:report": "vue-cli-service i18n:report --src \"./src/**/*.?(js|vue)\" --locales \"./src/locales/**/*.json\"",
+    "lint:fix": "vue-cli-service lint --max-warnings 0"
   },
   "dependencies": {
+    "@types/node": "^16.11.7",
     "axios": "~0.21.1",
     "bodybuilder": "^2.4.0",
     "leaflet": "~1.7.1",
     "leaflet-gesture-handling": "~1.2.1",
     "leaflet.markercluster": "~1.4.1",
     "material-design-icons-iconfont": "~6.1.0",
     "vue": "^2.6.13",
+    "vue-i18n": "^8.26.3",
+    "vue-i18n-bridge": "^9.2.0-beta.10",
```

Comment:
vue-18n is the library we use for all translations. If you want to see it in use, check src/i18n.ts
I also added a file as a snapshot
![image](https://user-images.githubusercontent.com/74095263/145858685-4d8da2c5-da8c-4811-8912-770656bcd967.png)

Reply:
I meant the i18n-bridge.
From the [official Docs](https://vue-i18n.intlify.dev/guide/migration/ways.html)

> vue-i18n-bridge is a bridge to make the upgrade as easy as possible between vue-i18n@v8.26.1 or later and vue-i18n@v9.x.

Since we are not upgrading we shouldn't need this.

Reply:
we removed the i18n-bridge.

Reply:
we removed the i18n-bridge.